internal/pkg/cli/job_init.go

```diff
@@ -254,6 +253,26 @@ func (o *initJobOpts) RecommendActions() error {
 	return nil
 }
 
+func (o *initJobOpts) validateDuplicateJob() error {
+	_, err := o.store.GetJob(o.appName, o.name)
+	if err == nil {
+		log.Errorf(`It seems like you are trying to init a job that already exists.
+To recreate the job, please run:
```

Comment:
I wonder if we should mention that the manifest will remain and they can opt to reuse it or not.

Reply:
That's a good idea!